test/unit/inbound/process-submitted-settlement.test.js

```diff
@@ -0,0 +1,137 @@
+const mockCommit = jest.fn()
+const mockRollback = jest.fn()
+const mockTransaction = {
+  commit: mockCommit,
+  rollback: mockRollback
+}
+
+jest.mock('../../../app/data', () => {
+  return {
+    sequelize:
+       {
+         transaction: jest.fn().mockImplementation(() => {
+           return { ...mockTransaction }
+         })
+       }
+  }
+})
+
+describe('process submitted settlement request', () => {
+  jest.mock('../../../app/inbound/get-existing-settlement')
+  const getExistingSettlement = require('../../../app/inbound/get-existing-settlement')
+
+  jest.mock('../../../app/inbound/save-settlement')
+  const saveSettlement = require('../../../app/inbound/save-settlement')
+
+  const processSubmittedSettlement = require('../../../app/inbound/process-submitted-settlement')
```

Comment:
suggestion: move these outside the describe and group with other mocks for sequelize, etc